src/sgame/sg_bot_skilltree.cpp

```diff
@@ -24,6 +24,39 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
 #include "sg_bot_util.h"
+#include "../shared/parse.h"
+
+static std::set<std::string> bg_allowedSkillset;
+
+static std::set<std::string> BG_ParseAllowedSkillsetList( const std::string &behaviorsCsv )
+{
+	std::set<std::string> skills;
+
+	for (Parse_WordListSplitter i(behaviorsCsv); *i; ++i)
+	{
+		skills.insert( *i );
+	}
+
+	return skills;
+}
+
+static void BG_SetAllowedSkillset( Str::StringRef behaviorsCsv )
+{
+	bg_allowedSkillset = BG_ParseAllowedSkillsetList( behaviorsCsv );
+}
```

Comment:
I would recommend writing a `G_UpdateSkillsets` function that would loop over every bot and set back their skill level to the same level (using `BotSetSkillLevel( bot, bot->botMind->botSkill.level )`).

And then you can call it in this (`BG_SetAllowedSkillset`) function.

This would make the change reflected in the already added bots.